siliconcompiler/schema/schema_cfg.py

```diff
@@ -879,24 +869,59 @@ def schema_datasheet(cfg, design='default', name='default', mode='default'):
             schelp="""Pin reset value specified on a per mode basis. Legal reset
             values include weak1, weak0, strong0, strong1, highz.""")
 
-    # DC levels
-    metrics = {'vol': ['low output voltage level', (-0.2, 0, 0.2), 'V'],
+    # Electrical Specifications
+
+    # DC levels and Metrics
+    metrics = {'vsupply': ['supply operating voltage', (0.2, 0.3, 0.9), 'V'],
+               'vmax': ['absolute maximum voltage', (0.2, 0.3, 0.9), 'V'],
+               'vol': ['low output voltage level', (-0.2, 0, 0.2), 'V'],
                'voh': ['high output voltage level', (4.6, 4.8, 5.2), 'V'],
                'vil': ['low input voltage level', (-0.2, 0, 1.0), 'V'],
                'vih': ['high input voltage level', (1.4, 1.8, 2.2), 'V'],
                'vcm': ['common mode voltage', (0.3, 1.2, 1.6), 'V'],
                'vdiff': ['differential voltage', (0.2, 0.3, 0.9), 'V'],
+               'voffset': ['offset voltage', (0.2, 0.3, 0.9), 'V'],
                'vnoise': ['random voltage noise', (0, 0.01, 0.1), 'V'],
                'vhbm': ['HBM ESD tolerance', (200, 250, 300), 'V'],
                'vcdm': ['CDM ESD tolerance', (125, 150, 175), 'V'],
                'vmm': ['MM ESD tolerance', (100, 125, 150), 'V'],
-               'rdiff': ['differential pair resistance', (45, 50, 55), 'ohm'],
-               'rpullup': ['pullup resistance', (1000, 1200, 3000), 'ohm'],
-               'rpulldown': ['pulldown resistance', (1000, 1200, 3000), 'ohm'],
+               'rdiff': ['differential pair resistance', (45, 50, 55), 'Ohm'],
+               'rup': ['pullup resistance', (1000, 1200, 3000), 'Ohm'],
+               'rdown': ['pulldown resistance', (1000, 1200, 3000), 'Ohm'],
+               'rweakup': ['weak pullup resistance', (1000, 1200, 3000), 'Ohm'],
+               'rweakdown': ['weak pulldown resistance', (1000, 1200, 3000), 'Ohm'],
+               'rin': ['input resistance', (1000, 1200, 3000), 'Ohm'],
+               'rdrive': ['pin resistance', (1000, 1200, 3000), 'Ohm'],
+               'power': ['power consumption', (1, 2, 3), 'W'],
+               'isupply': ['supply currents', (1e-3, 12e-3, 15e-3), 'A'],
                'idrive': ['drive current', (10e-3, 12e-3, 15e-3), 'A'],
                'iinject': ['injection current', (1e-3, 1.2e-3, 1.5e-3), 'A'],
+               'ishort': ['short circuit current', (1e-3, 1.2e-3, 1.5e-3), 'A'],
+               'ioffset': ['offset current', (1e-3, 1.2e-3, 1.5e-3), 'A'],
+               'ibias': ['bias current', (1e-3, 1.2e-3, 1.5e-3), 'A'],
                'ileakage': ['leakage current', (1e-6, 1.2e-6, 1.5e-6), 'A'],
-               'capacitance': ['capacitance', (1e-12, 1.2e-12, 1.5e-12), 'F']}
+               'capacitance': ['capacitance', (1e-12, 1.2e-12, 1.5e-12), 'F'],
+               # Metrics
```

Comment:
AC Metrics?

Reply:
?

Reply:
Just wondering if the Metrics should have been AC Metrics

Reply:
aha, that might have been a comment confusion. There isn't ac/dc, just parameters with relpin and without relpin needs.